server/routes/overview/overviewController.ts

```diff
@@ -7,16 +7,18 @@ export default class OverviewController {
     req.session.createGoalForm = undefined
 
     const { prisonerSummary } = req.session
+    const { supportNeeds } = req.session
 
-    const view = new OverviewView(prisonerSummary, 'overview', prisonNumber)
+    const view = new OverviewView(prisonerSummary, 'overview', prisonNumber, supportNeeds)
     res.render('pages/overview/index', { ...view.renderArgs })
   }
 
   getSupportNeedsView: RequestHandler = async (req, res, next): Promise<void> => {
     const { prisonNumber } = req.params
     const { prisonerSummary } = req.session
+    const { supportNeeds } = req.session
 
-    const view = new OverviewView(prisonerSummary, 'support-needs', prisonNumber)
+    const view = new OverviewView(prisonerSummary, 'support-needs', prisonNumber, supportNeeds)
```

Comment:
Ah ... thats my bad from my last PR ....
I was not envisaging that `OverviewView` would need changing for this, as its the view for the Overview tab
In my last PR I created a new view class `SupportNeedsView` which I'm envisaging is used as the view for the support tab. Clearly I didnt wire it up in the controller! Sorry! My bad!

So .... I think you can revert your changes to `OverviewView` because it does not need to know about the support needs
And instead put the new properties for support needs in the `SupportNeedsView` class; then use it here on this line 21:
```
const view = new SupportNeedsView(prisonerSummary, 'support-needs', prisonNumber, supportNeeds)
```